plugins/modules/proxmox_cluster_join_info.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025, Florian Paul Azim Hoberg (@gyptazy) <florian.hoberg@credativ.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: proxmox_cluster_join_info
+version_added: 1.0.0
+short_description: Retrieve the join information of the Proxmox VE cluster
+description:
+  - Retrieve the join information of the Proxmox VE cluster.
+author: Florian Paul Azim Hoberg (@gyptazy)
+extends_documentation_fragment:
+  - community.proxmox.proxmox.actiongroup_proxmox
+  - community.proxmox.proxmox.documentation
+  - community.proxmox.attributes
+  - community.proxmox.attributes.info_module
+"""
+
+EXAMPLES = r"""
+- name: List existing Proxmox VE cluster join information
+  community.proxmox.proxmox_cluster_join_info:
+    api_host: proxmox1
+    api_user: root@pam
+    api_password: "{{ password | default(omit) }}"
+    api_token_id: "{{ token_id | default(omit) }}"
+    api_token_secret: "{{ token_secret | default(omit) }}"
+  register: proxmox_cluster_join
+"""
+
+RETURN = r"""
+proxmox_cluster_join:
```

Comment:
Why not use a shorter name, like `cluster` or `join` or `cluster_join`? This is the part the users *always* have to type, next to the registered variable's name.

Reply:
Fair enough, I just had a look at https://github.com/ansible-collections/community.proxmox/blob/main/plugins/modules/proxmox_storage_contents_info.py#L64 which uses `proxmox_storage_content`. I personally don't mind and we can change it. But imho, I like having the scope (`proxmox`) as a prefix or the whole module name which is unique. In the end, it's up to you. If you want to have it shorter, I'll change it.

Reply:
Why do you like having the scope here? This is not a top-level name, but something in a dictionary whose name is picked by the user.

If this would be a fact, you would be totally right. But this isn't a fact, but a key in a dictionary which only has other keys like `msg` and `changed`.